packages/apollo-gateway/src/index.ts

```diff
@@ -262,9 +267,8 @@ export class ApolloGateway implements GraphQLService {
       return;
     }
 
-    if (!('serviceDefinitions' in result)) return;
-
     if (
+      !result.serviceDefinitions ||
```

Comment:
Is this change intentional?

Reply:
Yep! See changes from our previous PR. The previous lines were required for TS, but we don't actually want the early return there + the types are now permitting after the typing changes we make in this PR.
 
https://github.com/apollographql/apollo-server/pull/3409/files#diff-7b5e66976a637efc823826a985fa8e65R319